webpack.dev.js

```diff
@@ -0,0 +1,117 @@
+const webpack = require('webpack');
+const path = require('path');
+const ManifestPlugin = require('webpack-manifest-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
+const exec = require('child_process').exec;
+const config = require('./gulp/config');
+
+const outputPath = path.resolve(`${config.outputPath}/${config.jsDirectory}`);
+const jsSource = `./${config.sourcePath}/${config.jsDirectory}`;
+const cleanPaths = [
+  `${config.outputPath}/fonts/*`,
+  `${config.outputPath}/images/*`,
+  `${config.outputPath}/stylesheets/*`,
+  `${config.outputPath}/javascripts/*`
+];
+const entries = {
```

Comment:
What can be done to avoid duplicating code across the three different webpack config files?

Reply:
we can export them from a single file and import them.